src/goto-instrument/goto_instrument_parse_options.cpp

```diff
@@ -1012,6 +1018,21 @@ void goto_instrument_parse_optionst::instrument_goto_program()
   else
     options.set_option("assert-to-assume", false);
 
+  if(cmdline.isset("use-abstraction"))
```

Comment:
Command line should be consistent with CBMC.

Reply:
Changed.